test/system_test.js

```diff
@@ -1,70 +0,0 @@
-'use strict';
```

Comment:
This is calling real (outside) service from a unit test and should be mocked. So I deleted it on my workspace to prevent internet access but I accidentally commit delete... I rollback